tests/lib/assertions/classic-model-24.assert

```diff
@@ -0,0 +1,44 @@
+type: model
+authority-id: test-snapd
+series: 16
+brand-id: test-snapd
+model: ubuntu-core-24-pc-amd64
+architecture: amd64
+base: core24
+classic: true
+distribution: ubuntu
+grade: dangerous
+serial-authority:
+  - generic
+snaps:
+  -
+    default-channel: 24/edge
+    name: pc
+    type: gadget
+  -
+    default-channel: 24/edge
```

Comment:
It should be 24/beta to make sure the kernel is signed with Canonical keys - not sure now if this is relevant for the tests where we use the assertion but better be safe.

Reply:
done